tests/pages/HomePage.test.ts

```
@@ -1,97 +1,77 @@
+			expect(wrapper.text()).toContain(p.tag);
+			expect(wrapper.text()).toContain(p.body);
 		});
+	});
 
+	it('renders the about section with the profile name', async () => {
+		const wrapper = mount(HomePage, { global });
 		await flushPromises();
+		profile.name
+			.toUpperCase()
+			.split(' ')
+			.forEach((part) => expect(wrapper.text()).toContain(part));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)